MIGRATING-0.6-0.7.md

```diff
@@ -331,10 +365,24 @@ It now also disallows out of bounds drawing by default. This behaviour can be ch
 
 ## Style module
 
-The `style` module has been removed.
+The `style` module has been removed. The items in it have been moved:
 
-TODO: Improve this section before release.
+- `PrimitiveStyle` and `PrimitiveStyleBuilder` are now available under `embedded_graphics::primitives::{PrimitiveStyle, PrimitiveStyleBuilder}`.
+- `TextStyle` and `TextStyleBuilder` were renamed are now available under `embedded_graphics::mono_font::{MonoTextStyle, MonoTextStyleBuilder}`.
+
+  Note that usage with `Text` has changed. See [the text changes section](#Text-rendering) for more.
+
+## Text and fonts
+
+The collection of builtin fonts are now sourced from public domain BDF fonts in the XOrg project. Due to this, they have slightly different dimensions and glyphs and so have changed names. Some sizes are not the same in the new set, but a rough mapping is as follows:
+
+- `fonts::Font6x6` -> Removed
+- `fonts::Font6x8` -> `mono_font::[ascii|latin1]::Font6x10`. If a display performs better when using 8px high fonts, `mono_font::[ascii|latin1]::Font5x8` is suggested instead.
```

Comment:
I'm unsure if we need to mention this here. This is very specific to some displays and also assumes that the text is vertically aligned to the 8px page borders. And many glyphs in the new 6x10 font are less than 8 px high, so that there might not be a performance difference at all.

As an alternative I would suggest to mention that the new fonts use higher bitmaps for the same cap height below the substitution table. And that drawing performance might be lower if the text isn't properly aligned with page borders, but for details about this we should refer to the driver docs.